### PR TITLE
Fix resnet missing layers.

### PR DESCRIPTION
**Pull request by qlzh727**
The official v1 model contains BN and Relu between input layer and pooling. See http://ethereon.github.io/netscope/#/gist/db945b393d40bfa26006. 

For v2, I am also convinced that it should have those 2 layers, based on Appendix: Implementation Details in https://arxiv.org/pdf/1603.05027.pdf. Here is the quote:

"When using the pre-activation Residual Units (Fig. 4(d)(e) and Fig. 5), we
pay special attention to the first and the last Residual Units of the entire network.
For the first Residual Unit (that follows a stand-alone convolutional layer,
conv1), **we adopt the first activation right after conv1 and before splitting into
two paths**; for the last Residual Unit (followed by average pooling and a fullyconnected
classifier), we adopt an extra activation right after its element-wise
addition. These two special cases are the natural outcome when we obtain the
pre-activation network via the modification procedure as shown in Fig. 5."

### CODE REVIEW

**Comment by irmowan** [comment]
You need to add `if` statement.
if version == 1:
   +bn+relu

For resnet_v2 without BN&ReLU. (first block will do it.)

**Comment by qlzh727** [comment]
Done. Thanks for the suggestion.

File: official/resnet/resnet_model.py [hunk 1]
```diff
@@ -504,6 +504,13 @@ def __call__(self, inputs, training):
           strides=self.conv_stride, data_format=self.data_format)
       inputs = tf.identity(inputs, 'initial_conv')
 
+      # We do not include batch normalization or activation functions in V2
```
[the remark this answers is not in the document]
Comment (qlzh727) [reply official/resnet/resnet_model.py]: Done.

**Comment by qlzh727** [comment]
Removed the doc for checkpoint, for now.

**Comment by irmowan** [comment]
Another Question: why not provide the accuracy of ImageNet with this model? you can compare it to the paper, as a way of checking net architecture or parameters.

**Comment by HiKapok** [comment]
looking forward to the updated pre-trained model, thank you.

**Comment by robieta** [comment]
@irmowan We removed the checkpoints (and corresponding accuracy estimates) as they are no longer valid with the new topology. They are currently being regenerated and the section of the readme that was deleted in this PR will be reinstated when they are ready.
